etc/kayobe/inventory/group_vars/seed/network-interfaces

```diff
@@ -14,16 +16,16 @@ bond0_net_bond_slaves:
 # Admin network to rest of control plane
 # NOTE: seed node is the default gateway for all other control nodes
 #       via this network, and all other nodes access this via 1GbE
-admin_oc_net_interface: "{{ bond0_net_interface }}.{{ admin_oc_net_vlan }}"
+admin_oc_net_interface: "{{ 1gbe_physical_interface }}.{{ admin_oc_net_vlan }}"
```

Comment:
This provides all compute with the default route, kinda like this on the bond

Reply:
That's a good point, maybe after we've got these new nodes provisioned we move this back and just keep the provisioning traffic on the 1GbE NIC